website/docs/styling-layout.md

```diff
@@ -101,6 +101,29 @@ function MyComponent() {
 }
 ```
 
+#### Theme Class Names
+
+`@docusarus/theme-common` provides some predefined CSS class names to provide access for developers to style layout of a page globally in Docusaurus. The purpose is to have stable classnames shared by all themes that are meant to be targeted by custom CSS.
```

Comment:
`@docusaurus/theme-common` is an implementation detail, not a public API currently, so we don't want to encourage users to import this code for now.  Just tell "We  provide predefined CSS  class  names..."

Reply:
Removed the reference to `@docusaurus-theme-common`